libs/guid/include/guid/oguid.h

```diff
@@ -0,0 +1,217 @@
+/*************************************************************************
+Simple namespace to wrap GUID generation
+Copyright (C) 2004 Microsoft Corporation
+*************************************************************************/
+#pragma once
+#ifndef MSO_PLATFORMADAPTERS_OGUID_H
+#define MSO_PLATFORMADAPTERS_OGUID_H
+
+#include <codecvt>
+#include <compileradapters/cppmacros.h>

```

Comment:
>compileradapters/cppmacros.h [](http://example.com/codeflow?start=10&length=28)

Make sure that all paths are matching to file names. Otherwise they fail to compile in non-Windows environments.